kamailio/default.cfg

```diff
@@ -621,7 +623,11 @@ route[PREPARE_INITIAL_REQUESTS]
 
 route[RELAY]
 {
-    if (isflagset(FLAG_INTERNALLY_SOURCED)) {
+    #!ifdef SIP_TRACE_ROLE
+    route(SEND_SIP_TRACE);
+    #!endif
+
+    if (isflagset(FLAG_INTERNALLY_SOURCED) || src_ip == myself) {
```

Comment:
@sergey-safarov why was this added ? ` || src_ip == myself`

Reply:
When messages is send from kazoo_apps to kamailio, this is first send to kamailio to own socket, and then to sip device.

Reply:
@sergey-safarov almost sure this will break existing stuff. if the reason was the `MESSAGE` role, then lets handle that differently in `HANDLE_MESSAGE`.

Reply:
Special handling for messages send kamailio to self is added.